pkg/endpoint/endpoint.go

```diff
@@ -233,6 +235,8 @@ func getKineStorageBackend(ctx context.Context, driver, dsn string, cfg Config)
 		backend, err = pgsql.New(ctx, dsn, cfg.BackendTLSConfig, cfg.ConnectionPoolConfig)
 	case MySQLBackend:
 		backend, err = mysql.New(ctx, dsn, cfg.BackendTLSConfig, cfg.ConnectionPoolConfig)
+	case JetStreamBackend:
+		backend, err = jetstream.New(ctx, dsn)
```

Comment:
Does Jetstream support TLS or connection pooling?

Reply:
TLS yes... connection pooling no

I should definitely expose the TLSConfig and revisit the connection logic

Reply:
For reference:
https://github.com/nats-io/nats.go#tls

Reply:
There is no need for connection pooling in JetStream many operations are multiplexed in parallel over the single connection (which can be TLS or un-encrypted) to (one of the) nats-server(s).

Reply:
TLS Support added